recipes/popt/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "popt" %}
+{% set version = "1.16" %}
+{% set sha256 = "e728ed296fe9f069a0e005003c3d6b2dde3d9cad453422a10d6558616d304cc8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://anduin.linuxfromscratch.org/BLFS/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libiconv
+  run:
+    - libgcc  # [linux]    
```

Comment:
We should be able to drop this as long as the recipe isn't using OpenMP or Fortran.

Reply:
Currently we rely on system provided libraries for things like `libstdc++` and the like. `libgcc` is only used if the `gcc` package is used, which normally occurs if OpenMP or Fortran is needed. Even then we try to be minimal about our requirements and use something like `libgfortran` to satisfy the Fortran requirement at run time.